docs/troubleshooting.md

```diff
@@ -143,25 +143,24 @@ often grub is used to load the kernel and set its command line.)
 
 ### AMD encoding latency issues
 If you notice unexpectedly high encoding latencies (e.g. in Moonlight's
-performance overlay) or strong fluctuations thereof, this is due to
-[missing support](https://gitlab.freedesktop.org/drm/amd/-/issues/3336)
-in Mesa/libva for AMD's low latency encoder mode. This is particularly
-problematic at higher resolutions (4K).
-
-Only the most recent development versions of mesa include support for this
-low-latency mode. It will be included in Mesa-24.2.
-
-In order to enable it, Sunshine has to be started with a special environment
-variable:
-
+performance overlay) or strong fluctuations thereof, your system's Mesa
+libraries are outdated (<24.2). This is particularly problematic at higher
+resolutions (4K).
+
+Starting with Mesa-24.2 applications can request a [low-latency
+mode](https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/30039) by
+running them with a special [environment
+variable](https://docs.mesa3d.org/envvars.html#envvar-AMD_DEBUG):
```

Comment:
```suggestion
Starting with Mesa-24.2 applications can request a
[low-latency mode](https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/30039)
by running them with a special
[environment variable](https://docs.mesa3d.org/envvars.html#envvar-AMD_DEBUG):
```

Usually I prefer not to break up the URLs in markdown.

Otherwise this PR looks fine to me.